### DISCUSSION TITLE
Offline Projects

### DISCUSSION THREAD

**Discussion by LilSefGaming** [ScratchAddons/ScratchAddons#6871] [Ideas] [answered]
Maybe there can be offline projects so we can continue using Addons in the browser. Because I really like clean up blocks+ but I don't like using the offline editor

**Comment by BroJac5246** [comment, marked as answer]
> Maybe there can be offline projects so we can continue using Addons in the browser.

That's probably not feasible because of storage limits browsers impose on extensions, among other reasons.
> Because I really like clean up blocks+ but I don't like using the offline editor

Would TurboWarp's offline editor work? It has our addons.

**Comment by LilSefGaming** [comment]
Didn't know that was a thing, thanks!
